README.md

```diff
@@ -395,6 +395,7 @@ export default {
   odd: // '{description} must be odd',
   even: // '{description} must be even',
   positive: // '{description} must be positive',
+  multipleOf: // '{description} must be a multiple of {multipleOf}',
```

Comment:
Please also update the https://github.com/ssured/ember-changeset-validations/blob/master/README.md#number section